src/content/en/fundamentals/performance/critical-rendering-path/measure-crp.md

```diff
@@ -5,20 +5,51 @@ description: Learn to measure the critical rendering path.
 {# wf_updated_on: 2014-09-17 #}
 {# wf_published_on: 2014-03-31 #}
 
-# Measuring the Critical Rendering Path with Navigation Timing {: .page-title }
+# Measuring the Critical Rendering Path {: .page-title }
 
 {% include "web/_shared/contributors/ilyagrigorik.html" %}
 
-You can't optimize what you can't measure. Thankfully, the Navigation 
-Timing API gives us all the necessary tools to measure each step of the 
-critical rendering path!
+The foundation of every solid performance strategy is good measurement and
+instrumentation. You can't optimize what you can't measure. This doc
+explains different approaches for measuring CRP performance.
 
-### TL;DR {: .hide-from-toc }
-- Navigation Timing provides high resolution timestamps for measuring CRP.
-- Browser emits series of consumable events which capture various stages of the CRP.
+The key difference between the different approaches is the different kind
+of metrics that they can capture.
```

Comment:
Nope. The verb should match the subject of the sentence, which is "the key difference". The way to test these scenarios is by removing the extra element ("the key difference... is").

This sentence is funky though so I'll reword it.